casper/src/test/scala/coop/rchain/casper/helper/RhoSpec.scala

```diff
@@ -30,9 +30,9 @@ object RhoSpec {
         } ++ Seq(
         SystemProcess.Definition[Task](
           "rho:io:stdlog",
-          Runtime.byteName(26),
+          Runtime.byteName(27),
           2,
-          26L,
+          27L,
```

Comment:
I think those numbers should've changed in the previous commit?
What happened before you updated those numbers?
It'd be nice to have a registration mechanism that doesn't require magic constants synced throughout the whole codebase

Reply:
done